src/Upgrader/Business/UpgraderFacade.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/**
+ * Copyright © 2021-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace Upgrader\Business;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class UpgraderFacade implements UpgraderFacadeInterface
+{
+    public const CODE_SUCCESS = 0;
```

Comment:
Can we move these constants into the AbstractCommand?

Reply:
++